include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
Never mind - I was able to reproduce this error on Windows; it should be fixed in the master. (cling repo sync will arrive in a minute.) I.e. I *believe* this part (and the `#include <vector>`) can be removed from the PR.